bundles/ranvier-commands/commands/config.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+module.exports = (srcPath) => {
+  const Broadcast = require(srcPath + 'Broadcast');
+  const Parser = require(srcPath + 'CommandParser').CommandParser;
+  const Data = require(srcPath + 'Data');
+
+  return {
+    usage: 'config <set/list> [setting] [value]',
+    command: (state) => (args, player) => {
+      args = args.trim();
+
+      if (!args.length) {
+        Broadcast.sayAt(player, 'Configure what?');
+        return state.CommandManager.get('help').execute('config', player);
+      }
+
+      const possibleCommands = ['set', 'list'];
+      const splitArgs = args.split(' ');
```

Comment:
Since you end up using up to three args below, and naming them, you could do something like this here:
```
const [ command, configToSet, valueToSet ] = args.split(' ');
```

Then, instead of checking for `splitArgs.length` you could check for `configToSet`.